kt-vytvor_db

```diff
@@ -17,9 +17,11 @@ sql_subory = [
 	'graficke-data.sql',
 ]
 
-# vytvorenie schem
-print 'CREATE SCHEMA kataster;'
-print 'SET search_path TO kataster,public;'
+# potrebne vymysliet tak, aby sa schema volala podla niecoho .. pripadne, aby bol nazov zadavany interaktivne
```

Comment:
Podla mna je na tomto mieste v poriadku natvrdo vyzadovat nazov schemy 'kataster'. Ak chces data triedit podla KU, mozes na to pouzit VIEW.